pwiz_tools/Skyline/CommandArgUsage.resx

```diff
@@ -928,4 +928,10 @@ greater peak area.</value>
   <data name="CommandArgs_ARG_FULL_SCAN_PRODUCT_ISOLATION_SCHEME_path_to_result_file_to_import_the_isolation_scheme" xml:space="preserve">
     <value>path/to/result-file to import the isolation scheme from</value>
   </data>
+  <data name="CommandArgs_GROUP_OTHER_FILE_TYPES" xml:space="preserve">
+    <value>Exporting other file types</value>
+  </data>
+  <data name="_exp_speclib_file" xml:space="preserve">
+    <value>Export a spectral library.</value>
```

Comment:
Let's make this: "Export experimental data in the current document to a spectral library file. Peak areas are used as spectrum intensities, peak apex times are used as retention times, and if the document has iRT values they will also be stored in the library."